test/e2e/serial/tests/workload_placement_nodelabel.go

```diff
@@ -211,6 +224,193 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
+
+			By("Verifing the NRT statistics are updated")
+			targetNodeNRTCurrent, err := e2enrt.FindFromList(nrtCandidates, targetNodeName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(e2enrt.CheckEqualAvailableResources(*targetNodeNRTInitial, *targetNodeNRTCurrent)).To(BeTrue())
+		})
+
+		Context("label two nodes with different label values but both matching the node affinity of the deployment pod of the test", func() {
+			var err error
+			var unlabelTarget, unlabelAlternative func() error
```

Comment:
let's make sure we call these functions if the tests fails for error, and thus exits before we call these functions in the expected place

Reply:
updated, thanks for highlighting that.